lib/postcodeapi/version.rb

```diff
@@ -1,3 +1,3 @@
 module Postcode
-  VERSION = "1.1.0"
+  VERSION = "2.0.0"
```

Comment:
I appreciate the gesture, but don't change the version number here, I'll handle that automatically when putting out a new release.